1 - identify organisations in the corpus/gather_texts.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>Two minor suggestions: </p><ol><li>Using <code>os.path.join()</code> instead of <code>../../..</code> to ensure reproducibility across OS</li><li>Using a <code>with </code> statement can make the code look cleaner e.g., </li></ol><pre class="ql-syntax" spellcheck="false">with open(../data-files/studies_on_water_scraped.json') as f:
    documents = json.load(f)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>